### ISSUE TITLE
Cannot Get Property Length

### ISSUE THREAD

**Issue by htmelvis** [linxtion/react-image-gallery#18]
I've noticed that I was receiving this error with react 0.13.3 : 
```Uncaught TypeError: Cannot read property 'length' of undefined```

I realized that it was not correctly getting the thumbnail count from this portion of the code: 

```
if (this.refs.thumbnails) {
      var thumbNode = this.refs.thumbnails;
      if (thumbNode.scrollWidth <= this.state.containerWidth) {
        return 0;
      }

      var totalThumbnails = thumbNode.children.length;

      // total scroll-x required to see the last thumbnail
      var totalScrollX = thumbNode.scrollWidth - this.state.containerWidth;

      // scroll-x required per index change
      var perIndexScrollX = totalScrollX / (totalThumbnails - 1);

      return indexDifference * perIndexScrollX;
    }
```

In turn I updated : 

``` var totalThumbnails = thumbNode.children.length``` TO: ``` var totalThumbnails = thumbNode.props.children.length``` 

This solves my issue. I don't know if you want that changed / or if you want to support 0.13.3

Thanks

**Comment by linxtion** [comment]
@htmelvis this is now fixed in `v0.5.0` of `react-image-gallery`

**Comment by htmelvis** [comment]
Yes, this fixes the error thank you!
